README.md

```diff
@@ -65,7 +70,7 @@ ens_process('Nick.ETH',
     do_beautify=True,
     # ens_tokenize()
     do_tokenize=True,
-    # the reason why the input is not normalized
+    # identify the reason why input may not be normalized
```

Comment:
Should be "is not normalized" because things like NFC normalization are reported as well (NFC is applied automatically so the name *can* be normalized).
Unless you meant "the likely but not certain reason" which also seems wrong.

Reply:
As for detecting if the name "is not" or "cannot be" normalized, that's what is_fatal is for.